core/workspace.ts

```diff
@@ -388,6 +389,12 @@ export class Workspace implements IASTNodeLocation {
    * @param newName New variable name.
    */
   renameVariableById(id: string, newName: string) {
+    deprecation.warn(
+      'Blockly.Workspace.renameVariableById',
```

Comment:
Do you think it's helpful to have this be `Blockly.Workspace.prototype.renameVariableById` for clarity? I'm honestly not sure whether people understand the prototype system or not, and as such whether this is more confusing or less confusing.

Reply:
Personally I find that mildly more confusing; I'm looking at these as basically just a brief pointer to save you hunting it down, and I think as-is they give enough direction.